qualabs/video-headline
# feature/plugin-fast-channel

## Description:
This pull request introduces a toggle feature for the Fast Channel integration, allowing users to enable or disable it through a flag.

## Related Issue (if appropriate):
We aim to add a configuration at the organization level to control the Fast Channel feature. Additionally, this pull request integrates Fast Channel as a dependency into video-headline through SSH.

## Steps to test change/Steps to reproduce bug:
1. Navigate to the 'plugin-fast-channel' branch in the video-headline repository.
2. Add the following configuration to the user's Django organization:
    {
      "fast-channel": {
        "enable": true
      }
    }
3. Ensure SSH keys are available to clone the repository.

## Screenshots (if appropriate):
